src/main/scala/part7science/ScienceServer.scala

```diff
@@ -60,7 +62,7 @@ object ScienceServer {
 
     // spinning up the server
     val kafkaProducer = getProducer()
-    val bindingFuture = Http().bindAndHandle(getRoute(kafkaProducer), "localhost", 9988)
+    val bindingFuture = Http().newServerAt("localhost", 9988).bind(getRoute(kafkaProducer))
 
```

Comment:
Symbol bindAndHandle is deprecated. Used Http().newServerAt(...)...bindFlow() to create server bindings.